modules/core/arrow-instances/src/main/kotlin/arrow/data/eithert_implicits.kt

```diff
@@ -80,26 +80,26 @@ object EitherTSemigroupKInstanceImplicits {
     }
 }
 
-inline fun <reified F, L> EitherT.Companion.functor(FF: Functor<F> = functor<F>()): Functor<EitherTPartialOf<F, L>> =
+fun <F, L> EitherT.Companion.functor(FF: Functor<F>): Functor<EitherTPartialOf<F, L>> =
         EitherTFunctorInstanceImplicits.instance(FF)
 
-inline fun <reified F, L> EitherT.Companion.applicative(MF: Monad<F> = monad<F>()): Applicative<EitherTPartialOf<F, L>> =
+fun <F, L> EitherT.Companion.applicative(MF: Monad<F>): Applicative<EitherTPartialOf<F, L>> =
         EitherTApplicativeInstanceImplicits.instance(MF)
 
-inline fun <reified F, L> EitherT.Companion.monad(MF: Monad<F> = monad<F>()): Monad<EitherTPartialOf<F, L>> =
+fun <F, L> EitherT.Companion.monad(MF: Monad<F>): Monad<EitherTPartialOf<F, L>> =
         EitherTMonadInstanceImplicits.instance(MF)
 
-inline fun <reified F, L> EitherT.Companion.applicativeError(MF: Monad<F> = monad<F>()): ApplicativeError<EitherTPartialOf<F, L>, L> =
+fun <F, L> EitherT.Companion.applicativeError(MF: Monad<F>): ApplicativeError<EitherTPartialOf<F, L>, L> =
         EitherTApplicativeErrorInstanceImplicits.instance(MF)
```

Comment:
Do we need the implicit objects any more? Instances could could be instantiated directly here. The implicit object was used to statically resolved by naming ocnventions the instance but since we are no longer depending on that runtime lookup in this branch we don't need that indirection and can instantiate the instance here directly if it makes sense.

Reply:
This code should be autogenerated now when doing @instance, I'm not sure why these aren't.